packages/view/backend/sirius-components-view-diagram-edit/src/main/java/org/eclipse/sirius/components/view/diagram/provider/DiagramDescriptionItemProvider.java

```diff
@@ -186,10 +185,6 @@ protected void collectNewChildDescriptors(Collection<Object> newChildDescriptors
         EdgeDescription edgeChild = DiagramFactory.eINSTANCE.createEdgeDescription();
         edgeChild.setName("Edge");
         EdgeStyle newEdgeStyle = DiagramFactory.eINSTANCE.createEdgeStyle();
-        FixedColor fixedColor = ViewFactory.eINSTANCE.createFixedColor();
```

Comment:
could it be possible to try to get the color palette of the view and get the first color of this palette, if such color exists?
(same for the edge)